zerver/tests/test_import_export.py

```diff
@@ -331,6 +338,14 @@ def verify_avatars(self, user: UserProfile) -> None:
         db_paths = {user_avatar_path(user) + ".original"}
         self.assertEqual(exported_paths, db_paths)
 
+    def verify_migration_status_file(self) -> None:
+        migration_status_fixture = "dummy_migrations_01.txt"
+        text = read_text("migration_status.txt")
+        fixture = self.fixture_data(
+            migration_status_fixture, "import_fixtures/migration_status_files"
+        )
+        self.assertEqual(text, fixture)
```

Comment:
Hmm, is this test going to fail if we add newer migrations?

Reply:
I don't think it will because when testing, `get_migration_status` is patched to return a fixture of migration status instead of generating a new one from the server. This is also done when testing the import side of things.